pkg/mock/fileUtils.go

```diff
@@ -249,3 +265,40 @@ func (f *FilesMock) Chdir(path string) error {
 	f.currentDir = strings.TrimLeft(path, f.Separator)
 	return nil
 }
+
+// Stat ...
+func (f *FilesMock) Stat(name string) (os.FileInfo, error) {
+
+	props, exists := f.files[f.toAbsPath(name)]
+
+	if !exists {
+
+		isDir, err := f.DirExists(f.toAbsPath(name))
+		if err != nil {
+			return nil, fmt.Errorf("Internal error inside mock: %w", err)
+		}
+		if !isDir {
+			return nil, &os.PathError{
+				Op:   "stat",
+				Path: name,
+				Err:  fmt.Errorf("no such file or directory"),
+			}
+		}
+
+		// we assume some default // in the free wild wia umask
+		var mode os.FileMode = 0755
+
+		props = &fileProperties{}
+		props.mode = &mode
+		props.content = &dirContent
+	}
+
+	return fileInfoMock{
+		// mock is inacurat here since we return the full path, not only the basename
+		// (e.g. 'dir/name.txt' vs name.txt). Can be improved ...
+		name:  path.Base(name),
```

Comment:
It looks like the comment above no longer applies. (?)

Reply:
... yes. Comment removed..